spinalcordtoolbox/scripts/sct_check_dependencies.py

```diff
@@ -235,6 +235,8 @@ def main(argv=None):
         os_running = 'linux'
     elif sys.platform.startswith('win32'):
         os_running = 'windows'
+    else:
+        os_running = 'unknown'
```

Comment:
This case was previously covered above, under `# initialization`, by setting the default value `os_running = 'not identified'`. I like your version better because a missing `else` branch is usually a red flag; it's nice to have all the logic for setting `os_running` together. But please remove the (now redundant) default initialization.

Reply:
Oh, good catch!! I definitely missed that.

Reply:
Both `e` and `install_software` seem to be 0-1 variables, where 1 indicates that some kind of failure was encountered. I don't think it's a great pattern, but given the current use, they do need to be initialized to zero.